### PR TITLE
🎨  YAY: Simplify date and authors markup

### PR DESCRIPTION
**Pull request by julien-c**
Companion PR on Hub side to come soon, but please already review/approve if you can

### CODE REVIEW

File: accelerating-pytorch.md [hunk 1]
```diff
@@ -1,29 +1,16 @@
...
+- user: juliensimon
 ---
 
-<h1>
-Accelerating PyTorch distributed fine-tuning with Intel technologies
-</h1>
-
-
-<div class="blog-metadata">
-    <small>Published November 19, 2021.</small>
```
Comment (LysandreJik) [inline accelerating-pytorch.md:12]: The published date will be automatically identified? Same with profile picture and name, will it be identified from the username?

Comment (LysandreJik) [reply accelerating-pytorch.md:12]: Ah or will this infer from the HF Hub account? :smiley:

**Review by LysandreJik — APPROVED** [review]
Love the change!

File: README.md [hunk 2]
```diff
@@ -31,29 +31,9 @@ thumbnail: /blog/assets/101_decision-transformers-train/thumbnail.gif
...
-        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1632748593235-60cae820b1c79a3e4b436664.jpeg?w=200&h=200&f=face" title="Gravatar">
-        <div class="bfc">
-            <code>ThomasSimonini</code>
-            <span class="fullname">Thomas Simonini</span>
-        </div>
-    </a>
-</div>
+{blog_metadata}
+
+{authors}
```
Comment (pcuenca) [inline README.md:36]: I think the instructions above about replacing the `href`, `src` and `span` need to change too.

**Review by pcuenca — COMMENTED** [review]
Awesome!

File: accelerating-pytorch.md (hunk 1 above)
Comment (julien-c) [reply accelerating-pytorch.md:12]: yes! to both

File: README.md (hunk 2 above)
Comment (julien-c) [reply README.md:34]: done in eb74f34b093b976677ce8d17052154409dbb94b7

**Review by coyotte508 — APPROVED** [review]

File: accelerating-pytorch.md (hunk 1 above)
Comment (osanseviero) [reply accelerating-pytorch.md:12]: Very cool! :fire:

**Review by osanseviero — APPROVED** [review]
Nice!

**Comment by julien-c** [comment]
For posterity, here's the quick n dirty script (very dirty code)  i used to extract authors into cleaner metadata: https://gist.github.com/julien-c/a1832a9da1aa7379467e9518d2b3d768

**Pull request merged**